### DISCUSSION TITLE
VirusTotal results

### DISCUSSION THREAD

**Discussion by teijiIshida** [Akascape/CTkDesigner-Support#20] [General]
Hi, I was checking your trial exe on VirusTotal and it shows these results: https://www.virustotal.com/gui/file/f2fc95b39cd5b3a1616d2527decb524d932fa7a92e40fdda7c3d3ff40c4a2698

Should I be worried?

**Comment by Akascape** [comment]
@teijiIshida You can't expect a full 0/74 score in virus total with exe files made with pyinstaller. 
- The whole app is compiled with pyinstaller only. 
- Popular antivirus vendors have not detected the file, only 7 vendors with different results/names.

If you are worried:
- Test the software in windows sandbox/virtual environment and see.
- Make a simple python app and compile it with pyinstaller, then upload to virus total and see the detection results. You will better understand, (false positive issue of pyinstaller because same internal files are being used in all types of exe files).

From my side, the app is clean. 
Still if you are worried too much, don't run it.

**Comment by teijiIshida** [comment]
Thanks for the explanation.

**Comment by teijiIshida** [comment]
I bought your software today. I know this is probably a false positive, but it's kind scary and doesn't inspire confidence. Maybe you should submit your files to Malwarebytes so they can whitelist it.

![ctkdesigner](https://github.com/Akascape/CTkDesigner-Support/assets/33733459/f3e91423-1db2-422c-a350-a4aae50c8cc6)

**Reply by Akascape** [reply]
@teijiIshida I have to contact the vendors manually in this case. It also depends on the number of downloads a file has. I will try to do something. 
This is a similar issue  https://forums.malwarebytes.com/topic/279703-false-positive-generictrojanmaliciousdds/

(they also used pyinstaller)

**Comment by Akascape** [comment]
@teijiIshida I have raised the issue here: https://forums.malwarebytes.com/topic/313151-agentspywarestealerdds-false-positive/

It is now fixed I guess.
No more detection:
![image](https://github.com/Akascape/CTkDesigner-Support/assets/89206401/429dfea0-7b75-482f-af8f-3f02778a9889)

**Comment by teijiIshida** [comment]
Awesome, thank you!

**Comment by Akascape** [comment]
I also tried to reduce more false positives in the new version(6.0), and now the score is 5/72 in virus total. 
https://www.virustotal.com/gui/file/4ed781546908f1de2249568fccdbf241b4f5ea3f6aa21f3e07b9849c4fc19ec0?nocache=1

I hope one day we can make it to 0/72 :) The 5 detections are shown in all pyinstaller apps, so I can't do anything right now.

**Comment by InfHorus** [comment]
I think you need to license your application with a OV certificate or EV but its a bit expensive/complex, though on my side besides Windows Smart screen nothing triggers (I don't use any external anti-virus)

**Reply by InfHorus** [reply]
By having a certificate you can keep your application undetected across versions without resubmitting

**Reply by Akascape** [reply]
@InfHorus Currently the certificates are very expensive for me. I also have to buy the hsm security key. 

Although false detections and smart screens are different. The unknown publisher warning message is shown in windows which I think doesn't affect much, because even with OV it will be shown but with my name on it. 

But if antivirus detects something false positive then the users are concerned. 

Even though I haven't signed it, most antiviruses are not detecting anything right now.
